cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -43,6 +47,23 @@ class YamlConfig(
         val directoriesToPull: List<String> = listOf()) {
 
 
+    private val storage = StorageOptions.newBuilder().setProjectId(projectId).build().service
+    private val bucketLabel: Map<String, String> = mapOf(Pair("flank", "apks"))
+    private val storageLocation = "us-central1"
+    private val daysToLive = 30
```

Comment:
by default we probably shouldn't set a delete rule. A user can add one manually if they want later. I think it'd be surprising if someone used flank and then realized the artifacts are auto deleted.